db/data/autotest_files/jupyter/student_files/submission.ipynb

```diff
@@ -0,0 +1,106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6920ccbe",
+   "metadata": {},
+   "source": [
+    "Welcome to your assignment! Do the questions and write the answers in the code cells provided below"
```

Comment:
Overall please make the same style changes I mentioned in the test notebook here as well.